src/gr11review/part2/Utility.java

```diff
@@ -1,5 +1,356 @@
 package gr11review.part2;
+import java.io.BufferedReader.*;
+import java.io.*;
+import java.util.*;
 
+/**
+* Grade 11 Review Part 2
+* @Author: V.Shek & K.M
+*
+*/
 public class Utility {
+ /**
+  * Outputs sum of numbers in input by adding all the integers in an input with a combination of string and integers
+  *
+  * @param str is the String variable that is being read with integers included 
+  * @return sum of numbers in input
+  * @author V.Shek
+  */
+ public static int sumNumbers(String str) {
+  int y = 0;
+  int z = 0;
+  for (int f4 = 0; f4 < str.length(); f4++) {
+    if (Character.isDigit(str.charAt(f4))) {
+      for (int f5 = f4; f5 < str.length(); f5++) {
+        if (!Character.isDigit(str.charAt(f5))) {
+          z = (Integer.parseInt(str.substring(f4, f5)));
+          y = y + z;
+          f4 = f5 - 1;
+          break;
+        }
+        if (f4 != f5) {
+          z = (Integer.parseInt(str.substring(f4, str.length())));
+          y = y + z;
+          break;
+      }
+      }
+    }
+  }
+  return y;
+}
+/**
+  * Outputs input with changes to the non alone integer find out the integer that's not alone and change it to the bigger one next to it
+  *
+  * @param int[] nums is the array size and int val is the input 
+  * @return not alone integer
+  * @author V.Shek
+  */
+public static int[] notAlone(int[] nums, int val) {
+ for (int i= 1;i<nums.length -1;i++){
+  if (nums[i] == val)
+  if (nums[i-1] != nums[i] && nums[i+1] != nums[i])
+   if (nums[i+1] > nums[i-1]) nums[i] = nums[i+1];
+   else if (nums[i+1] < nums[i-1]) nums[i] = nums[i-1];
+   }
+   return nums;
+}
+
+/**
+  * Outputs first word in the WordBank.txt according to alphebetical order
+  *
+  * @param read String words is the words present in the WordBank.txt file 
+  * @return words
+  * @author V.Shek
+  */
+public static String alphaWord(String words){
```

Comment:
@val method input should be a filename/path and the method should read words from the text file

-2